### PR TITLE
Update workshop outline

### PR DESCRIPTION
**Pull request by cmilica**

### CODE REVIEW

**Comment by agitter** [comment]
Can I revert the changes to `learning_objectives.md` here so we can use #54 to discuss those and this pull request for the outline?

**Comment by cmilica** [comment]
@agitter yes!

File: workshop-shell.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+- **Dataset**
+  - preprocessing
+  - samples, features, class labels
+  - model
+
+## Learning
+
+- Training set
+  - model selection
+  - hyperparameters
```
Comment (agitter) [inline workshop-shell.md]: I suggest we move the hyperparameters later.  First we would want to practice building a decision rule by hand and seeing how it performs on test data.  We can possibly move to evaluation metrics after that.

Once we've seen how to evaluate a model, we can introduce hyperparameters and validation sets to choose hyperparameters.  As noted here, the number of thresholds or splits allowed in the decision tree is the relevant hyperparameter.

An open question is how much to do by hand and how much to do in the ml4bio software.  Do we start off working entirely on paper with a small version of the dataset (20-50 instances?) and then move to the software to introduce its functionality and see how to repeat these same steps with the full dataset?  We'll need the software at some point when we move to random forests, logistic regression, and neural networks.

Comment (cmilica) [reply workshop-shell.md]: So, that would be the hyperparameters in the last part of the workshop, possibly even day 2? Or do we want to introduce the whole process with the T-cells? 

Yes, I would use "by hand" example as the very beginning, and as a method of developing a sense for the classifiers and ML. It also depends on the activities we have for the rest of the workshop.

Comment (agitter) [reply workshop-shell.md]: I was hoping to fit hyperparameters into day 1 toward the end.  We'll have to start blocking time for each of these sections of the outline to see if that is practical.

**Pull request closed without merging**